core/Debug.carp

```diff
@@ -28,16 +29,17 @@
                        (System.exit 1)))
                 ())))
 
-  ;; Print the value of an expression to stdout, then return its value.
+  (doc trace "Print the value of an expression to stdout, then return its value.")
   (defn trace [x]
     (do
       (IO.println &(str &x))
       x))
 
-  ;; Leak some memory. Useful for testing tools that detect leaks.
+  (doc leak-array "Leak some memory. Useful for testing tools that detect leaks.")
   (register leak-array (Fn [a] ()) "Debug_leak_MINUS_array")
 
   )
+)
```

Comment:
did you mean to add this bracket ?

Reply:
No, I didn’t. Wow, this was the error that made the compiler throw an opaque error message. Thank you for reading through this diff!

Reply:
No problem, I saw your reported issue above so decided it was worthwhile just in case - I got lucky :)